doc/lammps.1

```diff
@@ -226,6 +226,20 @@ arguments of the "dump" command. See the
 .B LAMMPS
 manual for details on either of the two commands.
 .TP
+\fB\-r2info <restart file> <keyword> ...\fR or
+\fB\-restart2info <restart file> <keyword> ...\fR
+Write infomation about the <restart file> previously written by
+.B LAMMPS
+to the screen and immediately exit.  Following <restart file>
+argument, additional keywords for the
+.B LAMMPS
+"info" command may be added to increase the amount of information
+written. By default "system" "group" "fix" "compute" are already
+set. See the
+.B LAMMPS
+manual for details on either of the two commands.
```

Comment:
I don't understand what "either of the two commands" means ?

Reply:
cut-n-paste error. Should be 'for details on the "info" command.'